src/main/java/edu/ucsb/cs48/s20/demo/entities/Student.java

```diff
@@ -57,12 +57,11 @@ public void setId(long id) {
     }
 
     public String getEmail() {
-        return this.email.replaceFirst("@umail.ucsb.edu", "@ucsb.edu");
+        return this.email;
     }
 
     public void setEmail(String email) {
-        
-        this.email = email;
+        this.email = email.replaceFirst("@umail.ucsb.edu", "@ucsb.edu");
```

Comment:
Should this line also be present when setting the email via the constructor?

Reply:
yes it should.  good catch